eng/common/scripts/Prepare-Release.ps1

```diff
@@ -109,7 +109,10 @@ else
   $ParsedReleaseDate = [datetime]$ReleaseDate
 }
 
-$releaseDateString = $ParsedReleaseDate.ToString("MM/dd/yyyy")
+# On some machines, where people modified date format via Windows Registry,
+# "$ParsedReleaseDate.ToString("MM/dd/yyyy")" produces the date as "MM-dd-yyyy", which is probably a bug somewhere,
+# yet if we concatenate the parts individually, everything works as expected.
+$releaseDateString = $ParsedReleaseDate.ToString("MM") + "/" + $ParsedReleaseDate.ToString("dd") + "/" + $ParsedReleaseDate.ToString("yyyy")
```

Comment:
```suggestion
$releaseDateString = $ParsedReleaseDate.ToSTring("MM/dd/yyyy", [CultureInfo]::InvariantCulture)
```
Can you try that on your machine to see if it works?

Reply:
Thank you! It works. It is a better solution.